[IMP] l10n_ph update on BIR2306 & 2307 (#18360)
odoo/documentation: content/applications/finance/fiscal_localizations/philippines.rst

   SLSP`.
#. In the :guilabel:`DAT Export Options` window, click :guilabel:`Export`.

.. _localizations/philippines/reports/bir2306_2307:

BIR 2306 & 2307
~~~~~~~~~~~~~~~

Odoo supports the generation of **BIR Form 2306** (Certificate of Final Tax Withheld at Source) and
**BIR Form 2307** (Certificate of Creditable Tax Withheld at Source) in latest BIR PDF format.

To access them, navigate to :menuselection:`Accounting --> Reporting --> Tax Report`, click
:icon:`fa-book` :guilabel:`Report`, and select :guilabel:`2306 & 2307`.

Based on the period filtered, Odoo displays the transactions subject to each :abbr:`EWHT (Expanded
Withholding Tax)` and :abbr:`FWHT (Final Withholding Tax)` grouped by ATC code per partner level.

To export the certificates into PDF:

#. Make sure to filter by the appropriate **Month** or **Quarter** using the calendar filter.
#. By default, Odoo exports certificates for all the partners. To select a partner, use the
   :guilabel:`Partners` filter.
#. Click the :guilabel:`PDF` button to export the BIR-compliant PDF certificate.

.. note::

>    If the export contains, Odoo gathers them in a single ZIP file.

Reviewer: ```suggestion
.. note::
   If the export contains multiple certificates, Odoo gathers them in a single ZIP file.
```

   SLSP`.
#. In the :guilabel:`DAT Export Options` window, click :guilabel:`Export`.

.. _localizations/philippines/reports/bir2306_2307:

BIR 2306 & 2307
~~~~~~~~~~~~~~~

> Odoo supports the generation of **BIR Form 2306** (Certificate of Final Tax Withheld at source) and

Reviewer: ```suggestion
Odoo supports the generation of **BIR Form 2306** (Certificate of Final Tax Withheld at Source) and
```
For consistency with the form 2307 :)

> **BIR Form 2307** (Certificate of Creditable Tax Withheld at Source) in latest BIR `PDF` format.

Reviewer: ```suggestion
**BIR Form 2307** (Certificate of Creditable Tax Withheld at Source) in the latest BIR PDF format.
```
Not necessary :)

> To access the reporting menu, navigate to :menuselection:`Accounting --> Reporting --> Tax Report`,

Reviewer: ```suggestion
To access them, navigate to :menuselection:`Accounting --> Reporting --> Tax Report`,
```

> click :icon:`fa-book` :guilabel:`Report`, and select :guilabel:`2306 & 2307`.

Reviewer 2: By default, these reports aren't accessible on a PH database. Do you know if you need some transactions for those to appear? Else, it might be worth documenting how to get these displayed.

I made some bills with withholding taxes and confirmed them but that didn't change anything.

<img width="1905" height="327" alt="Image" src="https://github.com/user-attachments/assets/27dae27d-3f93-46ac-86aa-7db986a87cca" />

Author: Uhm there is no any separate setting required to display the following report. It is part of the default report like 2550Q, SAWT &QAP, and etc.

I am not sure how you do not have other reports too hmm... it seems master, but is it an old DB? I doubt thogh.

Attaching the steps for your reference. https://drive.google.com/file/d/1sxCttALUVlXy6F4EG0jL-LBBfvg1Ieu6/view?usp=sharing

The runbot took so long to download the module, you may wanna fast-forward a bit.

Thank you!

Reviewer 2: Thank you for the recording! Looks good to me :)

Based on the period filtered, Odoo displays the transactions subject to each EWHT (Expanded Withholding

> Tax) and FWHT (Final Withholding Tax) grouped by ATC code per partner level.

Reviewer: ```suggestion
Based on the period filtered, Odoo displays the transactions subject to each :abbr:`EWHT (Expanded Withholding
Tax)` and :abbr:`FWHT (Final Withholding Tax)` grouped by ATC code per partner level.
```
We could use the [abbreviation markup](https://www.odoo.com/documentation/19.0/contributing/documentation/rst_guidelines.html#abbreviations) here

To export the certificates into PDF:

#. Make sure to filter by the appropriate **Month** or **Quarter** using the calendar filter.
#. By default, Odoo exports the certificates for all the partners applicable in the report. To

>    specifically choose a partner, filter using the :guilabel:`Partners` filter.

Reviewer: ```suggestion
#. By default, Odoo exports certificates for all partners. To
   select a partner, use the :guilabel:`Partners` filter.
```
Can be simplified

> #. Click the :guilabel:`PDF` button on the top-left corner of the screen to export the BIR-compliant

Reviewer: ```suggestion
#. Click the :guilabel:`PDF` button to export the BIR-compliant
```

   PDF certificate.

.. note::
   When exporting for a single partner, Odoo downloads an individual PDF file. If the export contains

>    multiple certificates for different partners, Odoo compiles them into a single `.ZIP` file.

Reviewer: ```suggestion
.. note::
   If the export contains multiple certificates, Odoo gathers them in a single ZIP file.
```
Can be simplified
